### PR TITLE
[SmartBits] Minor fixes and enhancements

### PR DESCRIPTION
**Pull request by rsetaluri**
* Fix xor operator
* Add repeat operator

### CODE REVIEW

**Review by leonardt — CHANGES_REQUESTED** [review]
We should add some tests to cover these

**Comment by rsetaluri** [comment]
> We should add some tests to cover these

Done

File: magma/smart/smart_bits.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def __or__(self, other: 'SmartExpr') -> 'SmartBinaryOp':
     def __xor__(self, other: 'SmartExpr') -> 'SmartBinaryOp':
         if not isinstance(other, SmartExpr):
             return NotImplemented
-        return SmartBinaryOp(operator.xor_, self, other)
+        return SmartBinaryOp(operator.xor, self, other)
```
Comment (leonardt) [inline magma/smart/smart_bits.py:72]: Is there a test covering xor?

Comment (rsetaluri) [reply magma/smart/smart_bits.py:72]: done

**Review by leonardt — APPROVED** [review]

**Pull request merged**